i18n/qqq.json

```diff
@@ -56,5 +57,10 @@
 	"tesseract-psm-12": "Form option for Tesseract page segmentation mode.",
 	"tesseract-psm-13": "Form option for Tesseract page segmentation mode.",
 	"tesseract-param-error": "Error message displayed when invalid values for Tesseract options are submitted.\n\nParameters:\n* $1 – the form label for the option. Currently, the only possible value here is {{msg-wm|wikimedia-ocr-tesseract-psm-label}}.\n* $2 – The value that was given.\n* $3 – the maximum value for the option (this will be an integer).",
-	"tesseract-internal-error": "Generic error message displayed when the tesseract command fails."
+	"tesseract-internal-error": "Generic error message displayed when the tesseract command fails.",
+	"transkribus-error-401": "Error Code '$1' :: The request is not authorized",
```

Comment:
The sentences in qqq.json are the help text to go along with the actual messages in en.json. They're shown to translators on TranslateWiki.net, who will use them to make decisions about how to translate things. So the need to explain what the message is for. See https://www.mediawiki.org/wiki/Help:System_message for some more info (this tool uses the pretty much the same system as MediaWiki).